project/AppDependencies.scala

```diff
@@ -7,22 +7,24 @@ object AppDependencies {
   lazy val dependencyOverrides = overrides
 
   private lazy val bootstrapVersion = "5.16.0"
```

Comment:
Please update to latest found in [the catalogue](https://catalogue.tax.service.gov.uk/repositories/api-scope)